.github/workflows/check_duplicates.yaml

```
@@ -1,6 +1,9 @@
+# Disabled workflow, kept for future re-enablement. Every line must stay
```

actionlint (1.7.12)

[error] workflow is empty

(syntax-check)